posthog/src/main/java/com/posthog/PostHog.kt

```diff
@@ -227,6 +242,23 @@ public class PostHog private constructor(
             getPreferences().setValue(DISTINCT_ID, value)
         }
 
+    private var isIdentified: Boolean
+        @JvmName("get-isIdentified")
+        get() {
+            var isIdentified: Boolean?
+            synchronized(identifiedLock) {
+                isIdentified = getPreferences().getValue(IS_IDENTIFIED) as? Boolean
+                if (isIdentified == null) {
+                    isIdentified = this.distinctId != this.anonymousId
+                    this.isIdentified = isIdentified as Boolean
```

Comment:
Since `reset` will delete the preferences, we can assume its `false` here, if there's a value its always the value itself (always true in the case)

Reply:
but what will happen for already identified users on the current sdk? if we assume it be `false` won't it return `false` even though they're identified.

Reply:
Good point, for back compatibility, this makes a lot of sense.
We should change it then to, if there's a value, use the value, if there's no value, we do the `this.distinctId != this.anonymousId` check, so we can eventually remove this fallback once the new SDK is widely adopted.

Reply:
I agree. I think I've done exactly that 😅